CONFIG_FORMAT.md

```diff
@@ -0,0 +1,177 @@
+# Guide to config file format
+
+The ncdfchecker tool is driven by a json file containing details of expected
+metadata entries and data values.
+
+An example json file is supplied with the tool as "example.json" this can be
+a useful starting point for developing your own driving file and is the easiest
+way to learn about the structure of the driving file. It is recommended you
+take a look at that first and then use this guide to understand what it is
+doing and how you could extend/alter it. It is assumed you are familiar with
+json files.
+
+For the most part, the structure of the driving file is freeform using the json
+dictionary format to define variables, their entries and the values associated
+with them. Loosely:
+
+```
+{
+  "some_fields_shortname" : {
```

Comment:
When I tried this the checker recognised the standard name rather than the short name. Maybe this should just say name to be more general?

Reply:
Changed to "name"